src/app/auth/auth.component.ts

```diff
@@ -71,6 +72,7 @@ export class AuthComponent implements OnInit, OnDestroy {
 
   ngOnInit() {
     this.loggingService.startSession();
+    this.logger.info('Initialising authentication window');
```

Comment:
This statement will be logged in both the main application, and the pop-up auth window.

I'm not sure if we need this statement actually, as the `AppComponent` already generates some logs when it is loaded in the pop-up window (and this already tells us that the redirect to our app occurred successfully).

The AppComponent generates these log statements:
```
AppConfig,[object Object]
Mode web
```

Reply:
Thanks for pointing it out, I think we can remove it!